source/includes/fact-minimum-versions.rst

```diff
@@ -0,0 +1,7 @@
+Starting in version 1.2, ``mongosync`` can sync between clusters running 
+older (pre-6.0) releases of MongoDB Server.  Use of this feature
```

Comment:
[nit]

Consider either "Using this feature..." or simply, "This feature requires...".

Reply:
Done